tests/job.go

```diff
@@ -17,30 +17,54 @@ import (
 func WaitForJobToSucceed(virtClient *kubecli.KubevirtClient, job *batchv1.Job, timeoutSec time.Duration) {
 	EventuallyWithOffset(1, func() bool {
 		job, err := (*virtClient).BatchV1().Jobs(job.Namespace).Get(job.Name, metav1.GetOptions{})
-		ExpectWithOffset(1, err).ToNot(HaveOccurred())
-		Expect(job.Status.Failed).NotTo(Equal(*job.Spec.BackoffLimit), "Job was expected to succeed but failed")
-		return job.Status.Succeeded > 0
+		ExpectWithOffset(2, err).ToNot(HaveOccurred())
```

Comment:
I don't see any change in behavior here. What is purpose of this?

Reply:
Sorry, I didn't mean the specific line. Did you try if the offset is correct?

Reply:
I tried with 2 tests that uses the success and fail scenarios and it was fine. Why do you suspect it to be wrong?

If you are asking in general why using the condition instead of the success and failure counters, then the answer is in the commit. The logic of success and failure is exposed through the condition in a simple manner, independent of the number of retry used or other backend behaviors. Using it is safer and logically simpler, fitting with other patterns that check conditions and not statistics.

Reply:
Thanks for checking, wanted to be sure that we get it right.